static/assets/js/feedback.js

```diff
@@ -1,3 +1,19 @@
+const notSTWords = [
+    /\bscratch\s*team\b/i,
+    /\bst\b/,
+]
+
+const punishmentWords = [
+    /\bban\b/,
+    /\bunban\b/,
+    /\b(please|pls|plz)\s*block\b/,
+    /\bkick\b/,
+    /\bpunish\b/,
+    /\bpunishment\b/,
+    /\bmute\b/,
+    /\bunmute\b/,

```

Comment:
Mute and unmute shouldn't be here alone since we have a project muting addon.

Reply:
Nice catch. I will try to do a `pls|plz|please`.